tests/testdata/python3/data/import_setuptools_pep660/__editable___example_0_1_0_finder.py

```diff
@@ -0,0 +1,68 @@
+import sys
```

Comment:
I guess this is copied from `setuptools`. Shall we provide a link in a module docstring?

Reply:
That's a good idea. They don't have seem to have any documentation on it though. Should we link to the file in setuptools that generates this code?

Reply:
https://github.com/pypa/setuptools/blob/main/setuptools/command/editable_wheel.py

Reply:
Yeah, preferably with a commit hash in the docstring as well.

Everything to make the `astroid` maintainer 10 years down the line what we were trying to achieve here :smile:

Reply:
Done!